modules/basics/src/main/java/com/opengamma/strata/basics/currency/MultiCurrencyAmountArray.java

```diff
@@ -56,14 +56,16 @@
     implements FxConvertible<CurrencyAmountArray>, ImmutableBean, Serializable {
 
   /**
+   * The size of this array.
```

Comment:
Actually I'm wrong here about the semantics - it used to be the size of the array which is in practice the number of scenarios. After your updates this is now the case everywhere, so I will merge.